src/components/MediaInput/index.ts

```diff
@@ -1 +1,2 @@
 export * from './MediaInput';
+export * from "./MediaInput.types";
```

Comment:
Not quite sure why there's a `"` and a `'` - do you have prettier running? This reminds me, we should add a Prettier pre-commit hook 😎